src/ajax.js

```diff
@@ -83,6 +82,8 @@ export function ajax(url, callback, data, options = {}) {
     }
 
     x.open(method, url);
+    // IE needs timoeut to be set after open
```

Comment:
Can we stick a #1410 in the comment somewhere?

I'd recommend this as a general process, whenever bugs get fixed... cause it's been super helpful in other work. Tickets accumulate more info than code comments, and can include things like screenshots... so they tend to me the most informative when you're later wondering why this code is the way it is.